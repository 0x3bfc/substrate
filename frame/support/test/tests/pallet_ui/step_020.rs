#[frame_support::pallet]
mod pallet {
	use frame_support::pallet_prelude::*;

	#[pallet::config]
	pub trait Config: frame_system::Config {}

	#[pallet::pallet]
	pub struct Pallet<T> {}

	#[pallet::interface]
	impl<T: Config> Interface for Pallet<T> {}

	#[pallet::call]
	impl<T: Config> Foo {}
}

fn main() {
}
