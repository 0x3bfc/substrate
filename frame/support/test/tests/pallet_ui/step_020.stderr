error: Invalid module type: expect `Module<T>` or `Module<T, I>`
  --> $DIR/step_020.rs:15:17
   |
15 |     impl<T: Trait> Foo {}
   |                    ^^^

error: expected `Module`
  --> $DIR/step_020.rs:15:17
   |
15 |     impl<T: Trait> Foo {}
   |                    ^^^
